user/TASK/chassis_task/chassis_task.h

```diff
@@ -28,21 +28,34 @@
 #define BACK_LEFT 2
 #define BACK_RIGHT 3
 
-#define MULTIPLIER 3
+#define SETPOINT_SENSITIVITY 5
 
-// RC channels
-#define RC_X 2
+// RC channels -- this indicated strafe drive
+// left stick: rotation
+// right stick: fwd/rev, left/right
+#define RC_X 0 
 #define RC_Y 1
-#define RC_Z 0
+#define RC_Z 2
+// For 
 
 //M3508 motors max and min CAN output
-#define M3508_MAX_OUT 1000
+#define M3508_MAX_OUT 10000
 #define M3508_MIN_OUT 50.0
+#define M3508_MAX_IOUT 40
 //M3508 speed PID constants
-#define M3508_KP 0.02
-#define M3508_KI 0.00
-#define M3508_KD 0.1
+#define M3508_KP 0.002f
+#define M3508_KI 0.00f
+#define M3508_KD 0.0f
+// Current Limiting Constants
+#define HYSTERESIS_PERIOD 5
+#define CURRENT_LIMIT 25000
```

Comment:
Do we know the unit of this number?

Reply:
Remains to be seen. It is the same as the current feedback from the motors.